packages/building-blocks/src/stories/__snapshots__/alerts.story.tsx.snap

```diff
@@ -14,21 +14,21 @@ exports[`Building-Blocks/Alert BasicAlert smoke-test 1`] = `
     <path d="M8 16a2 2 0 0 0 2-2H6a2 2 0 0 0 2 2zM8 1.918l-.797.161A4.002 4.002 0 0 0 4 6c0 .628-.134 2.197-.459 3.742-.16.767-.376 1.566-.663 2.258h10.244c-.287-.692-.502-1.49-.663-2.258C12.134 8.197 12 6.628 12 6a4.002 4.002 0 0 0-3.203-3.92L8 1.917zM14.22 12c.223.447.481.801.78 1H1c.299-.199.557-.553.78-1C2.68 10.2 3 6.88 3 6c0-2.42 1.72-4.44 4.005-4.901a1 1 0 1 1 1.99 0A5.002 5.002 0 0 1 13 6c0 .88.32 4.2 1.22 6z">
     </path>
   </svg>
-  <span class="Alert__AlertHeader-sc-2vlo3n-2 fpauiN">
+  <span class="sc-eqUAAy cHzsWi">
```

Comment:
We make use of the `Alert__AlertHeader` type classnames in some places. Is there any way to restore these?

Reply:
I actually don't know why this changed. I guess it has something to do with the styled components plugin for vite. Since this classname is dynamically generated by the build step there's no guarantee that it will be consistent. For example, in OTP-RR these classnames are generated by the vite/webpack configuration in otp-rr, unrelated to the vite config that generated this snapshot. So relying on it is not safe because it depends on the consumer of the component.